ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/router-internet-exit.j2

```diff
@@ -0,0 +1,51 @@
+{#
+ Copyright (c) 2023-2024 Arista Networks, Inc.
+ Use of this source code is governed by the Apache License 2.0
+ that can be found in the LICENSE file.
+#}
+{# doc - router internet exit #}
+{% if router_internet_exit is arista.avd.defined %}
+
+### Router Internet Exit
+
+{%     if router_internet_exit.exit_groups is arista.avd.defined %}
+#### Exit Groups Summary
+
+| Exit Groups | Local Connections | Fib Default |
+| ----------- | ----------------- | ----------- |
+{%         for exit_group in router_internet_exit.exit_groups %}
+{%             for local_connection in exit_group.local_connections | arista.avd.default([]) %}
+| {{ exit_group.name }} | {{ local_connection.name }} | - |
+{%             endfor %}
+{%             if exit_group.fib_default is arista.avd.defined(true) %}
+| {{ exit_group.name }} | - | True |
+{%             endif %}
+{%             if exit_group.local_connections is not arista.avd.defined and exit_group.fib_default is not arista.avd.defined(true) %}
+| {{ exit_group.name }} | - | - |
+{%             endif %}
+{%         endfor %}
+{%     endif %}
+
+{%     if router_internet_exit.policies is arista.avd.defined %}
```

Comment:
New line should be rendered only if required - what if you don't have a policy?
```suggestion
{%     if router_internet_exit.policies is arista.avd.defined %}

```

Reply:
Done!